BUILDING.md

```diff
@@ -96,37 +96,39 @@ When installing the SDK, the "Debugging Tools for Windows" feature must be enabl
 
 1. Setup the following:
 
-    * Python 2 for scripts in Chromium
-    * Python 3 for buildkit
-    * [MSYS2](https://www.msys2.org/) *(recommended, not required)* - MSYS2 provides both git and [quilt](https://savannah.nongnu.org/projects/quilt/); the latter provides a convenient system to apply and manage patches.
-        * After installing and updating all packages to the latest version, install git and quilt: `pacman -S git quilt`
+    * 7-zip
+    * Python 2.7 for scripts in Chromium
+    * Python 3.5+ for buildkit
 
-2. Make sure Python 2 is accessible in `PATH` as `python`.
+2. Make sure Python 2.7 is accessible in `PATH` as `python`.
 
 #### Setting up the buildspace tree and packaging files
 
 Setting up via CMD:
 
 ```
 mkdir buildspace\downloads
-py buildkit-launcher.py genbun windows
-py buildkit-launcher.py getsrc
-py buildkit-launcher.py subdom
-py buildkit-launcher.py genpkg windows
+python3 buildkit-launcher.py genbun windows
+python3 buildkit-launcher.py getsrc
+python3 buildkit-launcher.py subdom
+python3 buildkit-launcher.py genpkg windows
```

Comment:
The Python 3.5 and newer installers for Windows can install the `py` launcher. There's no `python3` command that is installed by the installer (AFAIK), and this can be confusing to users who copy these instructions verbatim.

From my experience, the `py` launcher will correctly read the shebang of `buildkit-launcher.py` and use Python 3 instead of Python 2 to run buildkit. If you're worried about this, you can just use `py -3` to force using Python 3.

Is there a reason why you chose to use `python3` here? If not, we could just explicitly tell users to ensure that they install the `py` launcher.

Reply:
I think you're right here. I just wanted to make it clear it was Python 3, but your explanation makes more sense